index.html

```diff
@@ -2,11 +2,16 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-    
```

Comment:
HTML not working, due to missing meta tag

Reply:
My bad, let me fix it

Reply:
Are you sure?